src/nnet3/nnet-nnet.h

```diff
@@ -179,6 +179,9 @@ class Nnet {
 
   /// returns index associated with this node name, or -1 if no such index.
   int32 GetNodeIndex(const std::string &node_name) const;
+  
+  /// rename the node name with specific index
+  void RenameNodeName(int32 node_index, std::string new_node_name);
```

Comment:
You could try to use const std::string &new_node_name as the argument.
Also this function could be called RenameNode